### PR TITLE
refactor: adjust devtool install

### PR DESCRIPTION
**Pull request by kimdre**

### CODE REVIEW

**Comment by kimdre** [comment]
@jkroepke I'm not quite sure how I could set the tools to a specific version inside tools.go

File: go.mod [hunk 1]
```diff
@@ -3,6 +3,7 @@ module github.com/germanbrew/terraform-provider-hetznerdns
 go 1.22.4
 
 require (
+	github.com/golangci/golangci-lint v1.59.1
```
Comment (jkroepke) [inline go.mod:6]: like here

**Review by jkroepke — APPROVED** [review]
> I could set the tools to a specific version inside tools.go

the specific version is set inside the go.mod

File: go.mod (hunk 1 above)
Comment (kimdre) [reply go.mod:6]: They are already there :D 

https://github.com/germanbrew/terraform-provider-hetznerdns/blob/f5c8801eda3c8659c9d96d757fed1a2881919a5e/go.mod#L6-L7

**Pull request merged**